flowbased-computation/flowbased-computation-impl/src/main/java/com/farao_community/farao/flowbased_computation/impl/FlowbasedComputationImpl.java

```diff
@@ -108,7 +108,7 @@ private void handleCurativeState(State state, Network network, Crac crac, ZonalD
 
         List<DataMonitoredBranch> dataMonitoredBranches = flowbasedDomain.findContingencyById(contingencyId).getDataMonitoredBranches();
         dataMonitoredBranches.forEach(dataMonitoredBranch -> {
-            if (dataMonitoredBranch.getInstantId().equals(afterCraInstantId)) {
+            if (dataMonitoredBranch.getInstantId().equals(afterCraInstant)) {
```

Comment:
sonar bug here (comparing string to instant)

Reply:
fixed